bin/land.js

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const childProcess = require('child_process');
+const https = require('https');
+const path = require('path');
+const util = require('util');
+const clipboardy = require('clipboardy');
+
+const args = process.argv.slice(2);
+const exec = util.promisify(childProcess.exec);
+const targetBranch = args[0] || 'master';
+const toolName = path.basename(process.argv[1]);
+const toolVersion = require('../package.json').version;
+
+const help = `\
+This is a Pull Request landing tool that automatically picks up commits from
+the branch adds metadata to them and merges them into the specified branch.
+
+Usage: ${toolName} <target-branch> [OPTION]
+       ${toolName} --help
+       ${toolName} --version
+
+Options:
+  --remote-name   get name from remote repo
+  --rebase        start interactive rebase of the source branch
+  --autosquash    move commits that begin with squash!/fixup! during rebase
+  --cherry-pick   apply commits from source branch onto <target-branch>
+  --clipboardy    print 'Landed in (...commitsHash)'
+  --help          print this help message and exit
+  --version       print version and exit
+`;
+
+const handleError = error => {
+  console.log(error);
```

Comment:
```suggestion
  console.error(error);
```

Reply:
I should change to console.log here there and everywhere?